code/lib/cli/src/generators/baseGenerator.ts

```diff
@@ -226,6 +226,7 @@ export async function baseGenerator(
   const addons = [
     '@storybook/addon-links',
     '@storybook/addon-essentials',
+    '@chromatic-com/storybook',
     ...(compiler ? [`@storybook/addon-webpack5-compiler-${compiler}`] : []),
     ...stripVersions(extraAddonsToInstall || []),
```

Comment:
see if it can be added once via `extraAddonsToInstall`?

Reply:
You should likely specify a major version to install, so that in the future, we don't install a newer version that might be incompatible.

Reply:
Good call on both. Fixed!